routes/api/v1/forecast.js

```diff
@@ -43,54 +43,57 @@ rp(options).then(body => {
     };
 
     rp(options2).then(body2 => {
-                    var o = {} // empty Object
-                    var key = "location";
-                    o[key] = request.query.location; // empty Array, which you can push() values into
-                    var second_key = "currently"
-                    o[second_key] = body2.currently
-                    var third_key = "hourly"
-                    o[third_key] = body2.hourly
+      var json_response = {} // empty Object
+      json_response["location"] = request.query.location; // empty Array, which you can push() values into
+      json_response["currently"] = body2.currently
+      json_response["hourly"] = body2.hourly
+      json_response["daily"] = body2.daily
 
-                    var third_key = "daily"
-                    o[third_key] = body2.daily
-                    // console.log(body2)
+      delete json_response.currently.time
```

Comment:
@fentontaylor could you take a look to see how I can better organize this code into different parts/folders? Also, what do you think of my approach to deleting things from the incoming JSON hash rather than save the data into a POJO first, kind of like what we were doing in mod 3